lib/vedeu/configuration/cli.rb

```diff
@@ -93,7 +93,8 @@ def configuration
             options[:drb] = true
           end
 
-          opts.on('--drb-host', 'Set the hostname/IP for the DRb server.') do |hostname|
+          opts.on('--drb-host',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.